code/modules/exosuits/premade/combat.dm

```diff
@@ -0,0 +1,80 @@
+/mob/living/exosuit/premade/combat
+	name = "combat exosuit"
+	desc = "A sleek, modern combat exosuit."
+
+/mob/living/exosuit/premade/combat/Initialize()
+	if(!arms)
+		arms = new /obj/item/mech_component/manipulators/combat(src)
+		arms.color = COLOR_DARK_GUNMETAL
+	if(!legs)
+		legs = new /obj/item/mech_component/propulsion/combat(src)
+		legs.color = COLOR_DARK_GUNMETAL
+	if(!head)
+		head = new /obj/item/mech_component/sensors/combat(src)
+		head.color = COLOR_DARK_GUNMETAL
+	if(!body)
+		body = new /obj/item/mech_component/chassis/combat(src)
+		body.color = COLOR_DARK_GUNMETAL
+
+	. = ..()
+
+/mob/living/exosuit/premade/combat/spawn_mech_equipment()
+	..()
+	install_system_initialize(new /obj/item/mech_equipment/mounted_system/taser(src), HARDPOINT_LEFT_HAND)
+	install_system_initialize(new /obj/item/mech_equipment/mounted_system/taser/ion(src), HARDPOINT_RIGHT_HAND)
+
+/obj/item/mech_component/manipulators/combat
+	name = "combat arms"
+	exosuit_desc_string = "flexible, advanced manipulators"
+	icon_state = "combat_arms"
+	melee_damage = 5
+	action_delay = 10
+	power_use = 50
```

Comment:
Should all the components defined in /premade go into files in /components that sort by component type, so it's easier to compare the different options?

Reply:
Sorted